lib/rspec/core/formatters/helpers.rb

```diff
@@ -30,8 +30,8 @@ def self.format_duration(duration)
                       end
 
           if duration > 60
-            minutes = (duration.to_i / 60).to_i
-            seconds = duration - minutes * 60
+            minutes = (duration.round / 60).to_i
+            seconds = (duration - minutes * 60).abs
```

Comment:
Why is the `abs` needed?  I'm trying to imagine a case where the expression in parens returns a negative value.